internal/campaigns/executor.go

```diff
@@ -60,12 +60,21 @@ type TaskStatus struct {
 	FinishedAt time.Time
 
 	// TODO: add current step and progress fields.
```

Comment:
is this todo comment related to the newly added line? If not, I'd add a newline here to avoid confusion

Reply:
I'll let @LawnGnome decide. He added the TODO and I wasn't sure whether he had something else in mind, like

```go
TotalSteps int
CurrentStep int
```

or something like that.